src/virtual/utils/reload-content-scripts.ts

```diff
@@ -33,6 +50,37 @@ export async function reloadContentScriptMv3(contentScript: ContentScript) {
     await browser.scripting.registerContentScripts([{ ...contentScript, id }]);
   }
 
+  await reloadTabsForContentScript(contentScript);
+}
+
+export async function reloadRuntimeContentScriptMv3(
+  contentScript: ContentScript,
+) {
+  logger.log('Reloading content script:', contentScript);
+  const registered = await browser.scripting.getRegisteredContentScripts();
+  logger.debug('Existing scripts:', registered);
+
+  const existing = registered.find((cs) => {
+    const hasEveryJs = contentScript.js?.every((js) => cs.js?.includes(js));
+    const hasEveryCss = contentScript.css?.every(
+      (css) => cs.css?.includes(css),
+    );
```

Comment:
This is a really bad way of checking this... extract the function and add unit tests